draft-dukhovni-tls-dnssec-chain.xml

```diff
@@ -901,6 +902,27 @@ _dane443.node1.provider.example. IN TLSA 1 1 1 ...
       extension are described in <xref target="pinning"/>.
     </t>
 
+    <t>
+      Each zone in the DANE chain validation path for a domainname can
+      impersonate the server to any client.  A mitigation is for clients
+      to prefer or require TLSA usage 2, binding a WebPKI issuer as the
+      issuer of the server's end-entity certificate -- this allows the
+      client to benefit from certificate transparency (CT) <xref
+        target="RFC6962"/>.  A less practical / less likely mitigation
+      is for clients that can to separately lookup the DANE chain up to
+      the last zone delegation identified in the TLS DNSSEC chain
+      extension, and verify that they match.
+    </t>
+
+    <t>
+      The DNS community may wish to develop a DNSSEC equivalent to
+      PKIX's CT.  Because DNSSEC has strong name constraints, CT for
+      DNSSEC need only log changes in signed delegations (both, changes
+      to unsigned, as well as DS RRset changes), which are much less
+      frequent than WebPKI end-entity certificate issuance or TLSA RRset
+      signing.
```

Comment:
I'd like to hear Paul's thoughts on this...

Reply:
There are thoughts about CT-DNSSEC, but I don't think this doument should or need to reference it.